package.json

```
@@ -20,7 +20,8 @@
         },
         "./dist/vue-select.css": {
             "import": "./dist/vue-select.css",
-            "require": "./dist/vue-select.css"
+            "require": "./dist/vue-select.css",
+             "style": "./dist/vue-select.css"
```

```suggestion
            "style": "./dist/vue-select.css"
```

I'd remove a space for a bit more code style consistency